<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>776</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QPushButton" name="btn_play">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>46</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>20</y>
       <width>46</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>15</y>
      <width>426</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;</string>
    </property>
    <property name="title">
     <string>Device Connection</string>
    </property>
    <widget class="QWidget" name="led_connect_status" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #C21807;
border: 1px solid black;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_connect_status">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>30</y>
       <width>276</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>No device connected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_connect_change">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>25</y>
       <width>75</width>
       <height>36</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>95</y>
      <width>546</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;</string>
    </property>
    <property name="title">
     <string>Cycle-Car Monitor</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Gyroscope</string>
     </property>
     <widget class="QLabel" name="lb_data_00">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_01">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_02">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>25</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Z-Axis</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>125</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Accelerometers</string>
     </property>
     <widget class="QLabel" name="lb_data_03">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_04">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_05">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>25</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Z-Axis</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>225</y>
       <width>251</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Active Suspension Motor</string>
     </property>
     <widget class="QLabel" name="lb_data_06">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_07">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encoder Position</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encoder Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_08">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>85</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Setpoint</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_09">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>105</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>85</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Measured</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>165</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>165</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>145</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>145</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FET Temperature</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>445</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Estimates</string>
     </property>
     <widget class="QLabel" name="lb_data_13">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceleration Angle</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Motor Acceleration</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>285</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>GPS</string>
     </property>
     <widget class="QLabel" name="lb_data_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_21">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Latitude</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Longitude</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>125</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>PID Controller</string>
     </property>
     <widget class="QLabel" name="lb_data_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_18">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_19">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>25</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>176</x>
        <y>25</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>105</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>85</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Actuator Torque</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>25</y>
       <width>251</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Speed</string>
     </property>
     <widget class="QLabel" name="lb_data_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_data_15">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>45</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
border: 1px solid #D2D2D2;</string>
      </property>
      <property name="text">
       <string>0.00</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Left Wheel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>25</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Right Wheel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>95</y>
      <width>666</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px;</string>
    </property>
    <property name="title">
     <string>Live Plots</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>65</y>
       <width>656</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>656</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_add_plot">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>86</width>
       <height>36</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Add Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_plot_2">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>25</y>
       <width>86</width>
       <height>36</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>X-Axis</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionoi">
   <property name="text">
    <string>oi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
